web/src/components/PoiPanelNavigation.tsx

```diff
@@ -21,19 +23,28 @@ const Label = styled.span`
   font-size: clamp(0.55rem, 1.6vh, ${props => props.theme.fonts.hintFontSize});
 `
 
-const Icon = styled.img`
+const Icon = styled.img<{ direction: string }>`
   width: 16px;
   height: 14px;
   flex-shrink: 0;
   padding: 0 8px;
   object-fit: contain;
   align-self: center;
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
+      transform: rotate(180deg);
```

Comment:
Why not the following as well?
```suggestion
      transform: scaleX(-1);
```